### ISSUE TITLE
UTF-8 setting required before running git-gerrit

### ISSUE THREAD

**Issue by SilverTux** [andrust/git-gerrit#2]
If I'm using python3 the following error occurs:

```
Traceback (most recent call last):
  File "./git-gerrit", line 16, in <module>
    GerritTUI(cfg).start()
  File "/opt/git-gerrit/GerritTUI.py", line 152, in start
    self.mainloop.run()
  File "/usr/lib64/python3.6/site-packages/urwid/main_loop.py", line 286, in run
    self._run()
  File "/usr/lib64/python3.6/site-packages/urwid/main_loop.py", line 384, in _run
    self.event_loop.run()
  File "/usr/lib64/python3.6/site-packages/urwid/main_loop.py", line 788, in run
    self._loop()
  File "/usr/lib64/python3.6/site-packages/urwid/main_loop.py", line 816, in _loop
    self._entering_idle()
  File "/usr/lib64/python3.6/site-packages/urwid/main_loop.py", line 777, in _entering_idle
    callback()
  File "/usr/lib64/python3.6/site-packages/urwid/main_loop.py", line 572, in entering_idle
    self.draw_screen()
  File "/usr/lib64/python3.6/site-packages/urwid/main_loop.py", line 587, in draw_screen
    self.screen.draw_screen(self.screen_size, canvas)
  File "/usr/lib64/python3.6/site-packages/urwid/raw_display.py", line 850, in draw_screen
    self.write(l)
  File "/usr/lib64/python3.6/site-packages/urwid/raw_display.py", line 279, in write
    self._term_output_file.write(data)
UnicodeEncodeError: 'ascii' codec can't encode character '\u250c' in position 0: ordinal not in range(128)
```

The problem goes away when I specify LANG=en_US.UTF-8 environment variable before running the script.

It would be nice if I don't have to set environment variables to run the script.

**Comment by SilverTux** [comment]
I have a fix for this issue as a pull request, see #3

**Comment by andrust** [comment]
Pr merged.
